src/common/imageio_rawspeed.cc

```diff
@@ -267,7 +283,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed_sraw(dt_image_t *img, RawImage r, d
            * we need to copy data from only channel to each of 3 channels
            */
 
-          out[k] = MAX(0.0f, (((float)(*in)) - black) / scale);
+          out[k] = (float)*in / (float)UINT16_MAX;
```

Comment:
Note that this is dt_imageio_open_rawspeed_sraw(), not dt_imageio_open_rawspeed_raw()
We load them as 4x sizeof(float) - as any other non-raw image, like e.g. jpeg.